toolchain/base/value_store.h

```diff
@@ -120,14 +128,23 @@ class ValueStore<StringId> {
     return values_[id.index];
   }
 
+  auto Print(llvm::raw_ostream& out) const -> void { Print(out, 0); }
+  auto Print(llvm::raw_ostream& out, int indent) const -> void {
+    for (auto value : values_) {
+      out.indent(indent);
+      // TODO: Consider making this more robust for strings that need escaping.
+      out << "- \"" << value << "\"\n";
```

Comment:
Here's one we made earlier:
```suggestion
      out << "- \"" << llvm::yaml::escape(value) << "\"\n";
```
You'll need to `#include "llvm/Support/YAMLParser.h"`.

Reply:
Thanks, done